### ISSUE TITLE
Sort countries in picker by localized names

### ISSUE THREAD

**Issue by remy-shopopop** [kizitonwose/CountryPickerView#107]
Hello,
Nice library and very simple usage ;)
I have an issue (locale in French). Countries are correctly regrouped by localized name but the order inside is not correct sometimes (ie: "Allemagne" is after "Azerbaïdjan").
Do you think it will be possible to sort countries by localized names ?
Maybe by replacing : 
```
            groupedData.forEach{ key, value in
                groupedData[key] = value.sorted(by: { (lhs, rhs) -> Bool in
                    return lhs.name < rhs.name
                })
            }
```
with :
```
            groupedData.forEach{ key, value in
                groupedData[key] = value.sorted(by: { (lhs, rhs) -> Bool in
                    return lhs.localizedName() ?? lhs.name < rhs.localizedName() ?? rhs.name
                })
            }
```

Thanks a lot !
Rémy

**Comment by kizitonwose** [comment]
You are absolutely right, the sorting logic should use the localized name when available. I will make the necessary changes over the weekend. However, if this is very urgent for you, you can make the changes and send a PR.

**Comment by remy-shopopop** [comment]
Thanks @kizitonwose. I did send you the PR. Have a nice day !

**Comment by kizitonwose** [comment]
Thanks for the input. This is now released in [3.1.0](https://github.com/kizitonwose/CountryPickerView/releases/tag/3.1.0)
